### PR TITLE
[docs] remove legacy examples

### PR DESCRIPTION
**Pull request by davidiw**
what an exciting 6+ months...

this removes all the legacy examples... some won't work any more and others are broken and would require massaging..

note: the ts nft example is not there yet and moon coin needs to be ported over... but that's probably better than trying to make our legacy tutorials still ... work

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3528)

### CODE REVIEW

File: developer-docs-site/docs/tutorials/your-first-nft.md [hunk 1]
```diff
@@ -8,351 +8,483 @@ import TabItem from '@theme/TabItem';
...
+
+  Install the necessary dependencies:
+  ```
+  curl -sSL https://install.python-poetry.org | python3
+  poetry update
+  ```
+
+  Run the `transfer-coin` example:
+  ```sh
+  poetry run python -m examples.simple-nft
```
Comment (jjleng) [inline developer-docs-site/docs/tutorials/your-first-nft.md:63]: is this necessary? 🙄

File: developer-docs-site/static/examples/python/first_nft.py [hunk 2]
```diff
@@ -1,358 +0,0 @@
...
-        bob, alice.address(), alice.address(), collection_name, token_name
-    )
-    print(
-        f"Alice's token balance: {client.get_token_balance(alice.address(), alice.address(), collection_name, token_name)}"
-    )
-    print(
-        f"Bob's token balance: {client.get_token_balance(bob.address(), alice.address(), collection_name, token_name)}"
-    )
-
-    print("\n=== On-chain lazy mint through semi-fungible token ===")
```
Comment (areshand) [inline developer-docs-site/static/examples/python/first_nft.py:316]: Please keep the example

Comment (davidiw) [reply developer-docs-site/static/examples/python/first_nft.py:316]: Let's resurrect it in the python sdk. This isn't the right place for it.

File: developer-docs-site/sidebars.js [hunk 3]
```diff
@@ -27,14 +27,11 @@ const sidebars = {
...
-        "tutorials/first-transaction-sdk",
-        "tutorials/your-first-nft-sdk",
-        "tutorials/first-move-module-cli",
         "tutorials/first-transaction",
+        "tutorials/your-first-nft",
         "tutorials/first-move-module",
         "tutorials/first-dapp",
-        "tutorials/first-coin",
-        "tutorials/your-first-nft",
+        // "tutorials/first-coin",
```
Comment (jjleng) [inline developer-docs-site/sidebars.js:34]: intentional?

**Review by jjleng — APPROVED** [review]

File: developer-docs-site/docs/tutorials/your-first-nft.md (hunk 1 above)
Comment (davidiw) [reply developer-docs-site/docs/tutorials/your-first-nft.md:63]: Yes, we want to make the tutorial stand alone.

File: developer-docs-site/sidebars.js (hunk 3 above)
Comment (davidiw) [reply developer-docs-site/sidebars.js:34]: Yes, it needs to be updated

**Pull request merged**